### ISSUE TITLE
counter_isomorphic compile error: `hydrate` and `ssr` enabled as features

### ISSUE THREAD

**Issue by agilarity** [leptos-rs/leptos#916]
Run `cargo make test -p counter_isomorphic` from `leptos/examples` to reproduce.

**Comment by gbj** [comment]
I'm not sure what you're trying to achieve with this (there are no tests for that example). If you use `cargo make check` or something, it will avoid the mutually exclusive features that are defined in `Cargo.toml`. I'm not sure why `cargo make test` doesn't do that, but it never has if you run it from any of the sub-directories, as far as I know.
```toml
[package.metadata.cargo-all-features]
denylist = ["actix-files", "actix-web", "leptos_actix", "stable"]
skip_feature_sets = [["ssr", "hydrate"]]
``` 

You can run tests for the separate features separately (`cargo test --features=ssr`, `cargo test --features=hydrate`)

**Comment by agilarity** [comment]
Thank you for looking into this problem for me.

My apologies for not providing context.

The `cargo make test-examples` task runs tests for all examples. I expect it to run without error, even when no tests exist in a project. It ran without error before I created #904 but failed after I pulled in the latest changes. I thought the new feature checks that were added since then had uncovered problems in some examples.

I ran into the problem when I ran `cargo make test-examples`. I used `cargo make test -p counter_isomorphic` in my report to make it easy to verify a specific example.

I am surprised to learn that `cargo test` and `cargo make test` do not produce the same results. It is interesting that `cargo test` works without specifying any features. I wonder why `cargo make test` fails. I may look into this later.

Thank you for your analysis and kind feedback.
